aqua-examples/near-integration/README.md

```diff
@@ -3,6 +3,14 @@
 
 We provide integration examples for both a [Fluence JS](https://github.com/fluencelabs/fluence-js) node based on the [NEAR API JS](https://docs.near.org/docs/api/javascript-library) and distributed [Wasm services](https://github.com/fluencelabs/marine) wrapping the [NEAR RPC API](https://docs.near.org/docs/api/rpc). A [NEAR CLI](https://docs.near.org/docs/tools/near-cli) integration is planned for the near future.
 
+## Prerequisites
```

Comment:
given that that's the first time we're using it and are moving people off the aqua cli tool, that's not enough. the fluence cli explainer doesn't have to be here but it needs to thorough and, at least for some time, contrast fluence cli with aqua and marine cli

Reply:
Agree, makes sense.

Reply:
the tooling section is fine but a) too much for aqua,marine and b) misses fluence cli. (keep the section, add fluence cli and find a home for it either in the root dir or in the docs). in the meantime, i was more thinking about bridging the change to fluence cli. e.g., 

*For the purpose of this tutorial, we'll be using Fluence's new `fluence cli` tool, which unifies the cli's you have already been using, e.g.,  `aqua cli` and `marine cli`, and brings additional features such as bla bla bla. See the [fluence cli]() docs for more information.*

Reply:
yep, makes sense. I'll remove the tooling section from here. I also use its content in the overview for the "getting started with Fluence CLI" doc (drafting it now); it's probably a better context for it. And we can add either tooling only or the whole "getting started..." doc to the root or in the docs.